types/parameter/index.d.ts

```diff
@@ -0,0 +1,247 @@
+// Type definitions for parameter 3.6
+// Project: https://github.com/node-modules/parameter#readme
+// Definitions by: Innpaul Zhang <https://github.com/zhyupe>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.5
+
+export = parameter;
+
+declare class parameter {
```

Comment:
This is wrong, should start with Uppercase
https://github.com/node-modules/parameter/blob/master/index.js#L25
https://github.com/node-modules/parameter/blob/master/index.js#L139
This would require change in the namespace name as well

Reply:
Fixed in ca11aa9834e4ce4911f23f6d8ae181e351db499f